sass/MagicBox/MagicBox.scss

```diff
@@ -120,6 +120,9 @@
     }
     .magic-box-suggestions.magic-box-hasSuggestion {
       display: block;
+      div#coveo-magicbox-suggestions {
```

Comment:
maybe if you remove the `div` part here, the important in the other wouldn't be necessary (although maybe you will also have to be more specific)